conf/etcd/etcd.service.template

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Etcd Service
+
+[Service]	
+ExecStart=/usr/local/bin/etcd --config-file /etc/etcd/etcd.conf
+KillSignal=SIGTERM
+StandardOutput=append:/var/log/etcd.log
+StandardError=append:/var/log/etcd.err
```

Comment:
no this file is for setting up a linux service of etcd

Reply:
Mac has `/var/log` but that belongs to root. I figure it's worth mentioning in a short comment, that one can use this path on Mac but may need sudo. I figure the template file is most useful when one want to test and experience a feature, and that test may very likely happen on Mac instead of linux.